app/controllers/books_controller.rb

```diff
@@ -8,4 +8,13 @@ def index
   def show
     render json: Book.find(params[:id])
   end
+
+  def suggestions
+    result = BookSuggestionsByGenre.call(book_id: params[:book_id], user: current_user)
+    if result.success?
+      render json: result.books
```

Comment:
It would be nice to have pagination here. Too bad we don't have any pagination gem added to the project :/

Reply:
I have the pagination with `gem 'kaminari'`